gcloud/pubsub/topic.py

```diff
@@ -303,6 +303,31 @@ def set_iam_policy(self, policy, client=None):
             method='POST', path=path, data=resource)
         return Policy.from_api_repr(resp)
 
+    def test_iam_permissions(self, permissions, client=None):
```

Comment:
I realize the method is called `testIamPermissions` but wouldn't `check_iam_permissions` be a better name?